docs/tutorials/build_a_node_in_rust.md

```diff
@@ -727,15 +761,15 @@ let persister = YourDataPersister { data_dir: ldk_data_dir.clone() };
 
 **References:** [`Persister` docs](https://docs.rs/lightning-background-processor/*/lightning_background_processor/trait.Persister.html)
 
-### 18. Start Background Processing
+### Step 19. Start Background Processing
 
 **What it's used for:** running tasks periodically in the background to keep LDK operational
 
 **Example:**
 ```rust
 let background_processor = BackgroundProcessor::start(
 	persister,
-	handle_event_callback,
+	&invoice_payer,
```

Comment:
Currently, `EventHandler` is implemented for closures and `Arc<T>`, so would need to remove the reference here.

https://github.com/lightningdevkit/rust-lightning/blob/a5629e5ca2a7b089ac2a15f61f9549c2ebfcc608/lightning/src/util/events.rs#L1012-L1021

But an `Arc` would be preferable here because otherwise the user couldn't use `InvoicePayer` to make payments if passing it by value.